.github/workflows/gradle.yml

```diff
@@ -27,6 +27,10 @@ on:
         description: Test without DocumentDB? (1/0)
         required: false
         default: '1'
+      mavenFilesPreparation:
+        description: Prepare files to publish in maven repo? (1/0)
+        required: false
+        default: '0'
```

Comment:
default value is 0, which means we will not prepare files to publish by default?

Reply:
yes this does not need to be triggered by all prs. we  will trigger this on-demand when we have to create a release